resources/config-defaults.edn

```diff
@@ -22,6 +22,7 @@
  :smtp-host nil
  :smtp-port 25
  :mail-from nil
+ :public-url nil ;; url to rems SPA, for generating links in emails
```

Comment:
An assertion that `:public-url` ends with `/` would be good. Or automatic normalization. It can be done in `rems.config/env`'s `:start`.

Reply:
yeah that's a good idea